rollup.config.js

```diff
@@ -18,12 +20,17 @@ function createConfig(format) {
       commonjs(),
       replace({
         values: {
-          "process.env.NODE_ENV": JSON.stringify("production"),
+          "process.env.NODE_ENV": JSON.stringify(env),
         },
       }),
-      terser(),
-    ],
+      isProd && terser(),
+    ].filter(Boolean),
```

Comment:
I think this is actually unnecessary - rollup will automatically filter out falsy plugin values. See https://rollupjs.org/guide/en/#plugins